packages/paste-theme/src/styles/fonts.ts

```diff
@@ -0,0 +1,74 @@
+import {EmotionCSS} from '@twilio-paste/styling-library';
+
+export const pasteFonts = EmotionCSS`
+/* -------------------------------------------------------
+Variable font.
+*/
+@font-face {
+  font-family: 'Inter var';
+  font-weight: 100 900;
+  font-display: swap;
+  font-style: normal;
+  font-named-instance: 'Regular';
+  src: url("https://assets.twilio.com/public_assets/paste-fonts/main-1.2.0/Inter-roman.var.woff2?v=3.15") format("woff2");
+}
+@font-face {
+  font-family: 'Inter var';
+  font-weight: 100 900;
+  font-display: swap;
+  font-style: italic;
+  font-named-instance: 'Italic';
+  src: url("https://assets.twilio.com/public_assets/paste-fonts/main-1.2.0/Inter-italic.var.woff2?v=3.15") format("woff2");
+}
+/* --------------------------------------------------------------------------
+[EXPERIMENTAL] Multi-axis, single variable font.
+
+Slant axis is not yet widely supported (as of February 2019) and thus this
+multi-axis single variable font is opt-in rather than the default.
+
+When using this, you will probably need to set font-variation-settings
+explicitly, e.g.
+
+  * { font-variation-settings: "slnt" 0deg }
```

Comment:
I noticed we aren't setting this. Should we opt out of the experimental font for now? I'm not sure that we will need this.

Reply:
I am changing the italic slant slightly using the experimental font here https://github.com/twilio-labs/paste/pull/1045/files#diff-ec0ccc84914af419df413ea5920db554fc99334c8b9d9af6ff59a9250aa0acccR40 on line 40